src/config/dynamic-theme-fixes.config

```diff
@@ -935,6 +928,9 @@ CSS
     background-color: rgb(28, 30, 31) !important;
     color: darkgrey !important;
 }
+.jfk-bubble, .octotree-sidebar, .cc-pr__logo, .cc-octicon, #network canvas, img.network-tree {
+    filter: invert(94%) hue-rotate(180deg) contrast(90%) !important;
```

Comment:
Looks' better
![](https://i.imgur.com/MCqW1jd.png)

Reply:
invert(94.4%)

Reply:
That looks way better, nice!

Reply:
Idk how to put that into my pull request

Reply:
Go to changes then on the 3 dots of on right corner of the 'file' and then edit file

Reply:
That's basically what i did initially, when i fixed the issue, but @alexanderby just changed it to the INVERT section :D